bundle/Controller/UserController.php

```diff
@@ -149,6 +156,9 @@ public function editAction(
         Request $request
     ) {
         $user = $this->userService->loadUser($contentId);
+        if (!$this->permissionResolver->canUser('content', 'read', $user)) {
+            throw new CoreUnauthorizedException('content', 'read', ['userId' => $contentId]);
```

Comment:
```suggestion
            throw new CoreUnauthorizedException('content', 'edit', ['userId' => $contentId]);
```
Same on line above (seems we can't do multiline suggesions).